infra/main.bicep

```diff
@@ -120,5 +121,14 @@ module appServicePlan './core/host/appserviceplan.bicep' = {
   }
 }
 
+// Data outputs
+output AZURE_SQL_CATALOG_CONNECTION_STRING string = catalogDb.outputs.connectionStringKey
```

Comment:
output AZURE_SQL_CATALOG_CONNECTION_STRING_KEY
output AZURE_SQL_IDENTITY_CONNECTION_STRING_KEY


since it isn't the cstring it is the key